gradio/blocks.py

```diff
@@ -1258,6 +1260,7 @@ def launch(
         ssl_keyfile_password: str | None = None,
         quiet: bool = False,
         show_api: bool = True,
+        file_directories: List[str] = [],
```

Comment:
Better to [avoid using a mutable as a default](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/). Can you set this to be `None` and assign to it a list inside of the function?